CHANGELOG.md

```diff
@@ -16,10 +16,8 @@
 - `typography.$font-sizes` now expects a single Sass map of font-sizes. This is the base set of design tokens for your font-sizes, from which the responsive typographic scale is created.
 - Newly configured `line-height` utility classes to include the base `typography.$line-heights`
 - The `output-responsive-font-sizes` mixin expects a map data structure which includes `font-size` and `line-height`. If you were using the mixin you would have to change the parameter passed.
-
-### Changed
-
 - We now have a two-tier typography design token system. A single Sass map `$bitstylestypography-font-sizes` is used to create the existing responsive typography system for `h1` – `h6` elements, and the corresponding `.u-h1` – `.u-h6` responsive typography utility classes. The responsive typography for the base heading elements and the corresponding utility classes is built on top of these base design tokens. To update, all your unique font-sizes should be defined in the base scale in `settings/typography`, then referenced using `font-size.get()` to build your responsive typography scale in `settings/typography-responsive`.
+- Sizes scale naming has been updated to use to following convention `s,xs,2xl,...`
```

Comment:
It’s helpful to include a short note here on how users of bitstyles should change their codebase to deal with the change. Something like `If you already overrode the sizes with your own naming scheme, you don’t need to do anything. If you’re using the default naming scheme, replace all instances of `xxxl` in your codebase with `3xl`, `xxl` with `2xl`, `xxxs` with `3xs`, and `xxl` with `2xs`, wdyt?

Let’s also make the scale shown in the first sentence correct: `3xs, 2xs, xs, s, m, l, xl, 2xl, 3xl`